### DISCUSSION TITLE
پیشنهادات و ایرادات

### DISCUSSION THREAD

**Discussion by FreeLife2025** [hiddify/Hiddify-Manager#151] [Ideas ایدهها]
سلام 
خسته نباشید و دمتون گرم بابت زحماتتون
نرم افزار اندروید قابلیت اسکن qr نداره یا حداقل من پیدا نکردم و یکم هم کار کردن باهاش کنگه 
در مورد پنل اگر محدودیت اتصال اضافه بشه خیلی عالیه
در قسمت میزان حجم بشه از اعداد اعشاری استفاده کرد (برای اکانت های تست )
امکان شخصی سازی صفحه کابر از طریق  پنل ادمین 
اگر امکان تعویض سرور هم وجود داشته باشه عالی چون تو یه ساعاتی ای پی مستقیم سرور کار میده و نیازی به تانل نیست بشه خود کاربر انتخاب کنه به سرور ها هر کدوم سرعت بهتر بود با اون وصل بمونه 
راجع تونل ایران هم یه راهنمایی بکیند 

از چی استفاده کنیم سرعت و عملکرد بهتری داره مثلا iptables بهتره یا روش دیگه ای مد نطرتون هست 
و اینکه دوستان اگر تجربه سرور ایران خوب دارن لطفا به اشتراک بزارن 
این موضوع خیلی مشکل شده چون هم خیلی ترافیک گرون میگیرن و هم سرعت هاشون افتضاحه
ا

**Comment by FreeLife2025** [comment]
سلام من نصب جدید انجام دادم دامنه ست نمیشه و روی این صفحه میمونه 
با https کار نمیده ولی بدون https دامنه رو باز میکنه 

از کلود فلر استفاده کردم
ubuntu 20.4

**Comment by FreeLife2025** [comment]
![image](https://user-images.githubusercontent.com/125325796/218697254-b3e11fe8-8977-4ebd-834f-b4a2bbcb2a5a.png)

**Comment by hiddify-com** [comment]
برای relay کردن به آی پی های ایران و یا چندین سرور:
https://github.com/hiddify/hiddify-config/discussions/129
با فیلترشکن https کار میکنه؟

**Comment by FreeLife2025** [comment]
دامنه رو قبول کرد ولی سرعت خیلی کنده 
از هتزنر استفاده میکنم 

میخوام تونل هم داشته باشم 
read -p 'Enter your main server IP like(1.1.1.1): ' IP
echo "The main server IP is $IP"
sudo apt  install iptables iptables-persistent
sudo sysctl net.ipv4.ip_forward=1
sudo iptables -t nat -A POSTROUTING -p tcp --match multiport --dports  80,443 -j MASQUERADE 
sudo iptables -t nat -A PREROUTING -p tcp --match multiport --dports  80,443 -j DNAT --to-destination $IP
mkdir -p /etc/iptables/ 
iptables-save > /etc/iptables/rules.v4


این کد متوجه نشدم کجاش باید ای پی خارج بخوره کجاش ایران 
اگه میشه یه نمونه ای پی داخلش قرار بدید ما اون رو عوض کنیم ممنون
